Formula/c/cortexso.rb

```diff
@@ -0,0 +1,39 @@
+require "language/node"
+
+class Cortexso < Formula
+  desc "Drop-in, local AI alternative to the OpenAI stack"
+  homepage "https://jan.ai/cortex"
+  url "https://registry.npmjs.org/cortexso/-/cortexso-0.1.0.tgz"
+  sha256 "ad32120974c37b1e6bb8e745edf83ccf611045b38a21bafb8cd55fe2c638aeb9"
+  license "Apache-2.0"
+  head "https://github.com/janhq/cortex.git", branch: "dev"
+
+  depends_on "python-setuptools" => :build
+  depends_on "python@3.12" => :build
```

Comment:
do you remember why these were needed?

Reply:
@louis-jan I just opened https://github.com/louis-jan/cpuinfo/pull/1 to bump node-gyp to avoid needing setuptools

Reply:
Oops, that's awesome. Thanks @branchvincent. I will check this